internal/adapters/events/events_forwarder.go

```diff
@@ -138,6 +114,49 @@ func (*eventsForwarder) mergePlayerInfos(eventMetadata, fwdMetadata map[string]i
 	return m
 }
 
+func (f *eventsForwarder) buildRoomStatusMessage(eventAttributes events.RoomEventAttributes, forwarder entities.Forwarder) pb.RoomStatus {
+	event := pb.RoomStatus{
+		Room: &pb.Room{
+			Game:     eventAttributes.Game,
+			RoomId:   eventAttributes.RoomId,
+			Host:     eventAttributes.Host,
+			Port:     eventAttributes.Port,
+			Metadata: f.mergeInfos(forwarder.Options.Metadata, eventAttributes.Other),
```

Comment:
Hey @guilhermocc as `mergeInfos` is being used here could you move this function declaration to be before `mergeInfos` declaration?